From d9807188b372c706d657f3bfca0e9ae795e2dc04 Mon Sep 17 00:00:00 2001
From: Tao Wu <bjwt@linux.vnet.ibm.com>
Date: Tue, 19 Jul 2016 14:12:33 +0800
Subject: [PATCH] add several lines

---
 fstab | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/fstab b/fstab
index 9093bf1..530b21b 100644
--- a/fstab
+++ b/fstab
@@ -14,6 +14,18 @@
 
 /dev/mapper/luks-39c6fc8a-fe59-4e30-a556-8f35000cd377 /                       ext4    defaults,x-systemd.device-timeout=0 1 1
 abc
+abc
+abc
+abc
+abc
+abc
+abc
+abc
+abc
+abc
+abc
+abc
+abc
 UUID=a36e6738-ec0d-404c-b06d-f3036c307901 /boot                   ext4    defaults        1 2
 /dev/mapper/luks-666b64a6-f51d-4314-9831-c4f2045b91c2 /home                   ext4    defaults,x-systemd.device-timeout=0 1 2
 /dev/mapper/luks-3862f3c5-51ce-43ec-9fd8-257ea0216c39 /var                    ext4    defaults,x-systemd.device-timeout=0 1 2
-- 
1.8.4.2

